query_optimizer/ExecutionGenerator.cpp

```diff
@@ -714,7 +714,7 @@ void ExecutionGenerator::convertCopyFrom(
     const P::CopyFromPtr &physical_plan) {
   // CopyFrom is converted to a TextScan and a SaveBlocks.
 
-  const relation_id output_relation_id = physical_plan->catalog_relation()->getID();
+  const CatalogRelation *output_relation = physical_plan->catalog_relation();
```

Comment:
The original design of many classes have their getter methods return pointers other than references, e.g. the "physical::CopyFrom" class here. Perhaps at this moment we can just stick to the original design, and put an issue on uniforming the styles.